viper.py

```diff
@@ -11,6 +11,12 @@
 cve_list_output_json_file = 'cve_list.json'
 output_jsonl_file = 'data/vidata.json'
 
+if os.environ.get('API') is None:
+  API = 'api.kennasecurity.com'
+else:
+  API =  os.environ.get('API')
```

Comment:
You can supply a default value to `get`, so this could be simplified to just:
```
API = os.environ.get('API', 'api.kennasecurity.com')
```

Reply:
Good suggestion,  done.